bcda/cli_test.go

```diff
@@ -447,7 +453,68 @@ func (s *CLITestSuite) TestImportCCLFDirectory() {
 	assert.Contains(buf.String(), "Skipped 2 files.")
 	buf.Reset()
 
+	s.resetFiles("../shared_files/cclf_BadFileNames/")
+}
+func (s *CLITestSuite) TestCleanupCCLF() {
+	cclfmap := make(map[string][]*cclfFileMetadata)
+	s.setPendingDeletionDir()
+
+	// failed import: file that's within the threshold - stay put
+	acoID := "A0001"
+	fileTime, _ := time.Parse(time.RFC3339, "2018-11-20T10:00:00Z")
+	cclf0metadata := &cclfFileMetadata{
+		name:         "T.A0001.ACO.ZC0Y18.D181120.T1000011",
+		env:          "test",
+		acoID:        acoID,
+		cclfNum:      8,
+		perfYear:     18,
+		timestamp:    fileTime,
+		filePath:     "../shared_files/cclf/T.A0001.ACO.ZC0Y18.D181120.T1000011",
+		imported:     false,
+		deliveryDate: time.Now(),
+	}
+
+	// failed import: file that's over the threshold - should move
+	acoID = "A0001"
+	fileTime, _ = time.Parse(time.RFC3339, "2018-11-20T10:00:00Z")
+	cclf8metadata := &cclfFileMetadata{
+		name:         "T.A0001.ACO.ZC8Y18.D181120.T1000009",
+		env:          "test",
+		acoID:        acoID,
+		cclfNum:      8,
+		perfYear:     18,
+		timestamp:    fileTime,
+		filePath:     "../shared_files/cclf/T.A0001.ACO.ZC8Y18.D181120.T1000009",
+		imported:     false,
+		deliveryDate: fileTime,
+	}
+
+	// successfully imported file - should move
+	acoID = "A0001"
+	fileTime, _ = time.Parse(time.RFC3339, "2018-11-20T10:00:00Z")
+	cclf9metadata := &cclfFileMetadata{
+		name:      "T.A0001.ACO.ZC9Y18.D181120.T1000010",
+		env:       "test",
+		acoID:     acoID,
+		cclfNum:   9,
+		perfYear:  18,
+		timestamp: fileTime,
+		filePath:  "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010",
+		imported:  true,
+	}
+	cclfmap["A0001_18"] = []*cclfFileMetadata{cclf0metadata, cclf8metadata, cclf9metadata}
+	cleanupCCLF(cclfmap)
+
+	files, err := ioutil.ReadDir(os.Getenv("PENDING_DELETION_DIR"))
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Does using `log.Fatal()` here kill the whole run of all unit tests if there's an error? Might want to try https://godoc.org/github.com/stretchr/testify/require#FailNow

Reply:
Is this something we're going to implement in all of our testing code? Shooting for consistency here.